pdns/Makefile.am

```diff
@@ -1570,6 +1570,9 @@ dnsdist:
 if HAVE_SYSTEMD
 pdns.service: pdns.service.in
 	$(AM_V_GEN)sed -e 's![@]sbindir[@]!$(sbindir)!' -e 's![@]service_user[@]!$(service_user)!' -e 's![@]service_group[@]!$(service_group)!' < $< > $@
+if !HAVE_SYSTEMD_WITH_RUNTIME_DIR_ENV
+	$(AM_V_GEN)sed -e 's!/pdns_server!& --socket-dir=/var/run/pdns-%i!' -i $@
```

Comment:
Also noting it here: there is `%t` as variable for the runtime directory.

Reply:
Implemented